Answers.md

```diff
@@ -1,11 +1,19 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+React is a framework that exists between the code and the DOM. It tries to optimize the browser side by creating 
+a virtual DOM to compare changes made to the original and then render only those changes only to the DOM. It means you 
+can have more data on a page reloading in real time without bogging down the speed of the page.
 
 1.  What does it mean to _think_ in react?
+I don't know... perhaps to consider the break down of the page into components and what needs to be reuseable. 
 
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+The way I understand it is you write functional components until you need something to have it's own state. If 
+it needs to hold it's own data, then it needs to be in a class component.
 

```

Comment:
More or less, yes. Functional components are purely presentational. They have one job, and that is to return chunks of html to be rendered on the DOM.